Dockerfile

```
@@ -1,14 +1,16 @@
 FROM opensciencegrid/software-base:fresh
-LABEL maintainer "Lincoln Bryant <lincolnb@uchicago.edu>"
+LABEL maintainer "OSG Software <help@opensciencegrid.org>"
 
-RUN yum install -y yum-plugin-priorities
-RUN yum install -y osg-ca-certs osg-ce-bosco fetch-crl gratia-probes-cron openssh openssh-clients certbot
+RUN yum install -y osg-ce-bosco \
+                   openssh \
+                   openssh-clients \
+                   certbot && \
+    yum clean all --enablerepo=* && rm -rf /var/cache/yum/
```

```suggestion
    rm -rf /var/cache/yum/
```
Don't need both; one's enough.

---

Alternatively, `rm -rf /var/cache/*/*`.